pkg/kubewarden/l10n/en-us.yaml

```
@@ -274,6 +274,25 @@ kubewarden:
     url:
       banner:
         unavailable: The Policy Reporter UI proxy URL is unavailble, please ensure that the UI is properly configured.
+    headers:
+      label: Policy Reports
```

```suggestion
      label: Compliance
```


Opening here a discussion about renaming the "Policy Reports" tab to something else ("compliance" as said in the past?)

---

I like the idea, I've changed the wording to "Compliance".